packages/core/botpress/src/logger/db-transport.js

```diff
@@ -0,0 +1,51 @@
+import winston from 'winston'
+
+import { isEmpty } from 'lodash'
+
+import helpers from '../database/helpers'
+
+export default class DbTransport extends winston.Transport {
+  constructor(opts) {
+    super(opts)
+    this.name = 'DBLogger'
+    this.db = opts.db
+    opts.janitor.add({
+      table: 'logs',
+      ttl: opts.ttl
+    })
+  }
+
+  log(level, message, meta, callback) {
+    if (!isEmpty(meta)) {
+      message += ' ' + JSON.stringify(meta)
```

Comment:
Should re make it human-readable? i.e. `stringify(meta, null, 2)`  If not I would make the separator more computer-friendly like ` (meta=${json})`

Reply:
As logs are then exported to file I think keeping it on a single like makes more sense. I have actually not seen non-empty meta anywhere yet so will implement your 2nd suggestion.